js-wether/wether.js

```diff
@@ -0,0 +1,52 @@
+document.addEventListener("DOMContentLoaded", function () {
+    //API取得~HTML編集までの一連の動き
+    const wholeOparation = function () {
+        const xhr = new XMLHttpRequest();
+        const url = getWeatherApi();
+        xhr.open("GET", url, true);
+        xhr.onload = function () {
+            const dataJson = this.response;
+            const data = JSON.parse(dataJson);
+            const weatherEn = data.weather[0].main;
+            const weatherText = sharpData(weatherEn);
+            editHtml(weatherText);
+        };
+        xhr.send();
+    };
+
+    //APIへの問い合わせ
+    const getWeatherApi = function () {
```

Comment:
（修正必須）

こちら、コメントとメソッド名があってないです
（もしかしたら、APIという用語をきちんと理解できていなかもですね）

```suggestion
    const getWeatherApiEndPoint = function () {...}
```

とか

```suggestion
    const getWeatherApiUrl = function () {...}
```

ですね。

上部でメソッドを呼び出して値を代入していますが、基本的には左辺と右辺が同じものを指すようになっているか意識すると良いです。

```javascript
// これだと、左辺はURL、右辺はAPIなので、なんか別のものを代入しているように見える
const url = getWeatherApi();
```